ragstack-e2e-tests/e2e_tests/test_utils/astradb_vector_store_handler.py

```diff
@@ -201,23 +201,19 @@ def try_delete_with_backoff(self, collection: str, sleep=1, max_tries=5):
             self.try_delete_with_backoff(collection, sleep * 2, max_tries)
 
     def ensure_astra_env_clean(self, blocking=False):
-        logging.info("Ensuring astra env is clean")
```

Comment:
the problem with this approach is that if a collection deletion is in progress, the get_collections won't return it. so we could end up with the same issue we have now